### PR TITLE
Add Locals from Philadelphia And San Francisco Bay

### PR DESCRIPTION
**Pull request by MapGuy11**
Full Schedule

### CODE REVIEW

**Comment by MapGuy11** [comment]
Let Me Know Of Any Issues.

File: channels/us.m3u [hunk 1]
```diff
@@ -3570,7 +3586,3 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
 http://188.40.68.167/russia/disney/playlist.m3u8
 #EXTINF:-1 tvg-id="MeiGuoQieDaDianShi.us" tvg-name="美国切达电视" tvg-country="US" tvg-language="" tvg-logo="" group-title="",美国切达电视 (1080p)
 https://live.chdrstatic.com/cheddar/primary/4.m3u8
-#EXTINF:-1 tvg-id="nickjrtoo.us" tvg-name="Nick Jr. Too" tvg-country="UK;IE" tvg-language="English" tvg-logo="http://neczbm.to/images/channels/NickJrToo.png" group-title="Kids",Nick Jr. Too (576p)
-http://212.224.98.213:2200/EX/Nick_Jr_Too-uk/index.m3u8?token=
-#EXTINF:-1 tvg-id="nickjrtoo.us" tvg-name="Nick Jr. Too" tvg-country="UK;IE" tvg-language="English" tvg-logo="http://neczbm.to/images/channels/NickJrToo.png" group-title="Kids",Nick Jr. Too (576p)
-http://46.105.112.116/?watch=EX/Nick_Jr_Too-uk
```
Comment (freearhey) [inline channels/us.m3u:3576]: I guess you didn't intend to remove these links.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please put deleted link back so I can approve the rest of the changes.

File: channels/us.m3u (hunk 1 above)
Comment (MapGuy11) [reply channels/us.m3u:3576]: I can find what I removed is there a way to approve it without those changes

**Comment by MapGuy11** [comment]
I think I did it. Sorry

**Comment by Dum4G** [comment]
> I think I did it. Sorry

Go to patch-7 branch and fix everything that's needed. It will appear here automatically

**Comment by MapGuy11** [comment]
Now down sorry again!

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@MapGuy11 Thanks!

**Pull request merged**